soapui/src/main/java/com/eviware/soapui/SoapUI.java

```diff
@@ -809,7 +809,9 @@ public void run() {
                 }
 
                 if (isFirstLaunch && SoapUI.usingGraphicalEnvironment()) {
```

Comment:
Can we remove this code at all? The Endpoint Explorer is started by SoapUI:1014

Reply:
Yes, we can. I thought it made sense.